lib/screens/transfer/receive/receive_enter_data/interactor/receive_enter_data_bloc.dart

```diff
@@ -41,7 +39,7 @@ class ReceiveEnterDataBloc extends Bloc<ReceiveEnterDataEvents, ReceiveEnterData
       yield CreateInvoiceResultMapper().mapResultToState(state, result);
     } else if (event is ClearReceiveEnterDataState) {
       yield state.copyWith(
-        fiatAmount: 0.toString(),
+        fiatAmount: 0,
         isNextButtonEnabled: false,
         quantity: 0,
         seedsAmount: 0.toString(),
```

Comment:
left seedsAmount as string since we already have quantity - not sure why this code needs both